README.md

```diff
@@ -37,4 +37,14 @@ The Data flow:
     only on some kinds of the histograms. Some of the transformers add anomalies and change `y` and some only 
     add noise or trend.
     * The plots of changes are generated with usage of the `save_plot` function from `utils`.
-    * Everything is saved in the directory provided by the user. In our example `/some/directory`.
\ No newline at end of file
+    * Everything is saved in the directory provided by the user. In our example `/some/directory`.
+3. Download the data broken by every dimension with `download_broke.py`, eg:
+    ```bash
+    python canary/generator/download/download_broke.py /some/directory
+    ```
+4. Generate the broken data with anomalies with `generate_anomalies_in_broken_data.py`, eg:
```

Comment:
nit: same here, broken data might refer to a different concept :) Let's use "split" or find a better word for that.

Reply:
I think this comment is still open, the readme still refers to "broken data" at point 4.